Album.xcworkspace/xcuserdata/eyeray.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -7,336 +7,32 @@
       <BreakpointProxy
          BreakpointExtensionID = "Xcode.Breakpoint.FileBreakpoint">
          <BreakpointContent
-            uuid = "6403575F-542E-4595-A6EB-554F86371446"
+            uuid = "1EEF7E71-730C-4E3D-8044-E78CBA1CBDFD"
             shouldBeEnabled = "No"
             ignoreCount = "0"
             continueAfterRunningActions = "No"
-            filePath = "Album/ViewControllers/ProfileControllers/ProfileViewController.m"
```

Comment:
At some point we need to remove this .xbkptlist from the repo. 
We do not want reviewers to see this.